test/karma/stencil.config.ts

```diff
@@ -13,6 +13,7 @@ export const config: Config = {
   outputTargets: [
     {
       type: 'www',
+      empty: false,
```

Comment:
empty: true (which is the default) on the www target will remove the directory prior to a build being run. This stop that from happening. Important because I wanted the sibling build to empty the directory, since it's the first build task in this process. That way, nothing in previous build processes get removed and are accessible for tests.